src/tokens/saga.test.ts

```diff
@@ -12,99 +12,94 @@ import {
   watchAccountFundedOrLiquidated,
 } from 'src/tokens/saga'
 import { lastKnownTokenBalancesSelector } from 'src/tokens/selectors'
-import { fetchTokenBalancesFailure, setTokenBalances, StoredTokenBalances } from 'src/tokens/slice'
+import {
+  fetchTokenBalancesFailure,
+  setTokenBalances,
+  StoredTokenBalance,
+  StoredTokenBalances,
+} from 'src/tokens/slice'
 import { walletAddressSelector } from 'src/web3/selectors'
 import { createMockStore } from 'test/utils'
-import { mockAccount, mockCeurAddress, mockCusdAddress, mockTokenBalances } from 'test/values'
```

Comment:
when I added fixture data to `mockTokenBalances` it broke these tests-- even when I added balances of 0 for some tokens. There was an assumption that fixture data in this file would stay in sync with `mockTokenBalances`, otherwise a test would fail. I updated the code here to remove that assumption so we can add shared fixture data without breaking dependencies.